srml/executive/Cargo.toml

```diff
@@ -18,6 +18,7 @@ hex-literal = "0.2.0"
 substrate-primitives = { path = "../../core/primitives" }
 srml-indices = { path = "../indices" }
 balances = { package = "srml-balances", path = "../balances" }
+node-runtime = { path = "../../node/runtime" }
```

Comment:
Can't we avoid the entire `node-runtime` in tests and just use `test-runtime`? If not then maybe the tests should be placed elsewhere?

Reply:
We can but honestly, I am against it to a high extent. I really prefer testing executive (and a few other important pieces of the code with the __real runtime__).

Reply:
then create a new crate to do it in `/node/`. nothing in `/srml/` should be dependent on `/node/`. period.